Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1500 @@ label mas_wrs_twitch:
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_taskmanager:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Oh, is a program causing trouble? I hope I'm not the cause, ahaha..."
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_taskmanager')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_renpy",
+            category=['theren'pyvisualnovelengine'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_renpy:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "The place where it all began, ahaha!",
+           "Nice to see you taking an interest in this stuff."
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_renpy')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_solitaire",
+            category=['microsoftsolitairecollection'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_solitaire:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Oh, you like playing with cards? Nice!"
+           "You know, [player], I wouldn't mind playing a card game with you\nwhen I cross over. That'd be nice, don't you agree?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_solitaire')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_tiktok",
+            category=['tiktok'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_tiktok:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Hmmm, there's a lot of creative stuff being made here.\nDo you make content on here, [player]?", 
+           "If I ever got on here, I bet you'd follow me in a heartbeat. Ehehe~"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_tiktok')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_whatsapp",
```

Comment:
can probably make this generic to be for IM clients, like Messenger, Discord, TeamSpeak, etc.